app/controllers/membership_applications_controller.rb

```diff
@@ -68,6 +68,11 @@ def information
 
   end
 
+  def destroy
+    @membership_application.destroy
+    redirect_to membership_applications_url, notice: t('membership_applications.application_deleted')
```

Comment:
Not sure I follow this comment ... do we need to change the translation?  But, you say "it was correct ..." so I'm not sure what is meant here.